.github/workflows/freebsd.yml

```diff
@@ -31,7 +31,12 @@ jobs:
         uses: vmactions/freebsd-vm@v1
         with:
           prepare: |
-            pkg install -y gcc gmake
+            pkg install -y autoconf automake libtool
+            pkg install -y cmake ninja
           run: |
             MAKE=gmake ./configure
             gmake -j2 check || (cat tests/test-suite.log && exit 1)
```

Comment:
sorry I should've been clearer. `gmake` needs to be replaced with `make` throughout

Reply:
Ah, oops!